starlark/archive.go

```diff
@@ -30,6 +37,15 @@ func archiveFunc(thread *starlark.Thread, b *starlark.Builtin, args starlark.Tup
 		outputFile = "archive.tar.gz"
 	}
 
+	// Always include the script executed and the logs.
+	if script := thread.Local(identifiers.scriptName); includeScript && len(script.(string)) > 0 {
```

Comment:
nil check and conversion check needed; a test was failing because of the nil conversion. When running locally this wasnt a problem but the codepath must be slightly different w.r.t. to how the params are loaded/unpacked.